scalameta/semanticdb-scalac/src/test/scala/scala/meta/tests/SemanticSuite.scala

```diff
@@ -592,7 +592,7 @@ class SemanticSuite extends DatabaseSuite(SemanticdbMode.Slim) {
        |""".stripMargin
   )
 
-  sugars(
+  synthetics(
```

Comment:
I like this name very much. Thank you, @olafurpg, @ShaneDelmore and @gabro!

Reply:
I agree, very nice :)

Reply:
💯 

Be careful with the dryer 🤓